### ISSUE TITLE
Ask only once if I want to see Urban Dictionary results, or make it an argument

### ISSUE THREAD

**Issue by Spiritspeak** [r-lib/available#60]
It impedes workflow to keep asking whether urbandictionary results should be shown in available()
ideally there would be an argument like showUrbanDictionary=c(getOption("session.showUD",NA),T,F) that asks the user for their preference on first call, saves it in option "session.showUD" and then uses that choice unless the argument showUrbanDictionary is explicitly set to T or F

**Comment by jimhester** [comment]
I agree, the current behavior is not ideal, I agree only showing this dialog once would be ideal. PRs welcome!

**Comment by llrs** [comment]
This was addressed several years ago at https://github.com/r-lib/available/pull/50, but it missed the release, and there haven't been any release since them. You can use the GitHub/development version if you want to avoid this.
